debug.log

```diff
@@ -0,0 +1,3 @@
+[0923/123721.035:ERROR:registration_protocol_win.cc(103)] CreateFile: The system cannot find the file specified. (0x2)
```

Comment:
Is this file needed?

Reply:
I will check and remove if it is not needed